DEPLOYMENT.md

```diff
@@ -0,0 +1,3 @@
+# Deployment
+
+libp2p-stardust is integrated with the [libp2p Docker Hub](https://hub.docker.com/u/libp2p) organization. Docker containers are built for passing CI to the `master` branch and to `tags`. The libp2p infra team leverages these containers to deploy and host stardust services for users to test with. These servers should **not be relied on for Production applications**. If you wish to leverage stardust servers for your application, we recommend running your own as we currently do not guarantee uptime of these servers.
```

Comment:
"These servers"

It would be good to mention the addresses. Right now it mixes the existence of containers with these of the servers, maybe also separate the two sentences with more line breaks.

Reply:
I added a new line and made a change on `services`. Regarding the addresses I agree with you, and I will add them once the servers are deployed and I have the addresses